.gitignore

```diff
@@ -167,6 +168,8 @@ execution_time_of_individual_pytest_files.txt
 # Docs: Advance Examples MNIST data
 docs/advanced_examples/data/MNIST/
 
+# Docs: Advance Examples FHE training deployment files
+docs/advanced_examples/fhe_training
```

Comment:
Maybe we should target a tmp folder instead. This kind of practice leaves tons of circuit behind

Reply:
that's what I did, but if you don't run the full notebook until the cleanup part then some file stays

but I was planning to add a gitignore in the advances_examples directory instead, and put this there

Reply:
I've created a `.gitignore` in `advanced_examples`